site/kube-addon.md

```diff
@@ -18,22 +18,21 @@ The following topics are discussed:
 Weave Net can be installed onto your CNI-enabled Kubernetes cluster
 with a single command:
 
-* Kubernetes versions `1.6` and above:
-
-```
-$ kubectl apply -f https://git.io/weave-kube-1.6
-```
-
-* Kubernetes versions up to `1.5`:
-
 ```
-$ kubectl apply -f https://git.io/weave-kube
+$ kubectl apply -n kube-system -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
 ```
 
 After a few seconds, a Weave Net pod should be running on each
 Node and any further pods you create will be automatically attached to the Weave
 network.
 
+**Note:** In Kubernetes versions `1.6` and above, depending on how your cluster is configured, you may need to:
```

Comment:
As mentioned [here](https://weaveworks.slack.com/archives/C0DNGPKLY/p1494948825467088), I'm tempted to drop this "Note" altogether. @bboreham, WDYT?